tests/defaults.yml

```
@@ -8,6 +8,7 @@ parameters:
   facts:
     cloud: cloudscale
     sales_order: "10431"
+    service_level: "premium"
```

I'd use `guaranteed_availability` as the default service level for the tests, since that's the relevant value going forward.

```suggestion
    service_level: "guaranteed_availability"
```